pkg/database/database.go

```diff
@@ -122,6 +126,17 @@ func (db *Database) Open(ctx context.Context) error {
 	return db.OpenWithCacher(ctx, nil)
 }
 
+const driverName = "ocsql"
```

Comment:
I know this is what we had before, but while I was debugging this, I found a few bugs.

The registration should happen via an `init()` function at the package level:

```go
const driverName = "ocsql"

// Some docs
func init() {
  for _, v := range sql.Drivers() {
    if v == driverName {
      return
    }
  }
  ocsql.RegisterAllViews()
  sql.Register(driverName, ocsql.Wrap(&postgres.Driver{})
}
```

Reply:
Done, thanks for the input.